roles/v2k-bridge/templates/v2k_bridge_cluster.yaml.j2

```diff
@@ -19,6 +19,24 @@ spec:
       containers:
       - name: v2k-bridge
         image: dojot/v2k-bridge:{{ dojot_v2k_version}}
+        readinessProbe:
+          httpGet:
+            path: /ready
+            port: 9000
+          failureThreshold: 1
+          initialDelaySeconds: 5
+          periodSeconds: 5
+          successThreshold: 1
+          timeoutSeconds: 5
+        livenessProbe:
+          httpGet:
+            path: /live
+            port: 9000
+          failureThreshold: 3
+          initialDelaySeconds: 10
+          periodSeconds: 30
+          successThreshold: 1
+          timeoutSeconds: 5
```

Comment:
If the customer takes 5 seconds to respond doesn't that mean it is unhealthy?

Reply:
I took these values from the [lightship documentation](https://github.com/gajus/lightship/#kubernetes-container-probe-configuration), that is said to be a reasonable one. It is that high to prevent false-positives. I don't know if it would suggest it is unhealthy, since the service can be overloaded, so it would be slow to respond to the GET request.

Reply:
Ok, I understand that 1 second can generate false-positives, but wouldn't 3 seconds be an acceptable value? Does the service take 5 seconds to respond doest not indicates that it is unhealthy?